admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -59,7 +59,7 @@ $(document).ready(function() {
   /* validate price fields , as Thomas de Nabord said */
   $(".money-type input[type='text']").change(function validate(event) {
     var inputValue = $(this).val();
-    var parsedValue = truncateDecimals(parseFloat(inputValue),6);
+    var parsedValue = (new Number(inputValue)).toFixed(6);
```

Comment:
hmm, you may add the ``toFixed(6)`` inside the function, because this function is used multiple times everywhere in the world (and probably in the universe).

What do you think ?